server/interpreter/data/protos.go

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2017 Google Inc.
+ * https://github.com/NeilFraser/CodeCity
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package data
+
+// Protos is a struct containing prototypes for the various Object
```

Comment:
To me, Builtins would include (for example) Object, not just Object.prototype.  Protos carries only the prototypes, not any of the constructors (or libraries, like Math).